std/experimental/ndslice/slice.d

```diff
@@ -2659,15 +2659,17 @@ struct Slice(size_t _N, _Range)
         Increment `++` and Decrement `--` operators for a $(B fully defined index).
         +/
         auto ref opIndexUnary(string op)(Repeat!(N, size_t) _indexes)
-            if (op == `++` || op == `--`)
+            // @@@workaround@@@ for Issue 16473
+            //if (op == `++` || op == `--`)
         {
             mixin (`return ` ~ op ~ `_ptr[indexStride(_indexes)];`);
         }
```

Comment:
Unrelated to this review but there's a slightly better version of this:

```d
return mixin(op ~ `_ptr[indexStride(_indexes)]`);
```

It's because `mixin` may be either an expression or a statement.